go.mod

```diff
@@ -11,11 +11,10 @@ require (
 	github.com/go-sql-driver/mysql v1.6.0
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/mock v1.6.0
-	github.com/golang/protobuf v1.5.2
```

Comment:
I'm skeptical about this.

Reply:
It builds, so I guess we removed all the proto stuff when the tritter example went?